IndyAV/index.rst

```diff
@@ -0,0 +1,12 @@
+Indy AV
```

Comment:
Why not use this as an opportunity to talk about indyAV project and provide some links (like to the competition website / Dr. Crane's website)